extern/sector-storage/ffiwrapper/verifier_cgo.go

```diff
@@ -120,6 +122,11 @@ func (proofVerifier) VerifyAggregateSeals(aggregate proof5.AggregateSealVerifyPr
 	return ffi.VerifyAggregateSeals(aggregate)
 }
 
+func (proofVerifier) VerifyReplicaUpdate(update proof7.ReplicaUpdateInfo) (bool, error) {
+	v := ffi.FunctionsSectorUpdate{}
```

Comment:
I see conversation about this [here](https://github.com/filecoin-project/filecoin-ffi/pull/208#issuecomment-966457279), but I'm not sure why this breaks the pattern (why isn't the verify call in proofs.go like the others).

@ZenGround0 maybe you know?

Reply:
Yeah, we need to add those, see https://github.com/filecoin-project/filecoin-ffi/blob/master/proofs.go#L27 for example. (we want it there, so non-lotus things can use go types more easily without importing lotus)